### PR TITLE
About page

### PR DESCRIPTION
**Pull request by andylushman**
* Added about Page
* Added `about.yml` file
* Added About to nav menu
* Created routes for sub menu
* Added placeholders for 
     * Assets page
     * Faq page
     * Team page
* Modified `content_card` so that it now also takes `arrays` in addition to `strings` for the description.

* To get the project up and running I had to migrate the db which caused a changed to the `structure.sql`

### CODE REVIEW

File: app/views/site/about.html.slim [hunk 1]
```diff
@@ -0,0 +1,12 @@
+= render layout: 'layouts/shared/secondary_hero', locals: { title: 'About', sub_title: 'By the community, for the community', description: 'Bringing Denver Startup Week to life takes the support, skills, and drive of an entire community: entrepreneurs, investors, civic leaders, educators, business pioneers, and more.' } do
+  = render 'layouts/shared/about_secondary_hero_nav'
+
+div.Program-list-wrapper
```
Comment (scpaye506) [inline app/views/site/about.html.slim]: I know this is the same layout, but better if we don't re-use class names here

File: app/views/site/about.html.slim [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  = render 'components/content_card_list', attributes: { 'aria-label': "About page" }  do
+    - about_static_data[:items].each do |card|
+      = render 'components/content_card_list_item' do
+        = render 'components/content_card', card: card, title: "#{card[:title]}", page_url: "about/#{card[:button_page_url]}"
+
+= render layout: 'layouts/shared/layout_wrapper' do
+  div.Program-primary-callouts
```
Comment (scpaye506) [inline app/views/site/about.html.slim]: same here with the class name

File: db/structure.sql [hunk 3]
```diff
@@ -9,20 +9,6 @@ SET xmloption = content;
 SET client_min_messages = warning;
 SET row_security = off;
 
---
```
Comment (scpaye506) [inline db/structure.sql]: i know it did this automatically, but go ahead and revert the change in the file

Comment (andylushman) [reply db/structure.sql]: Removed that changes.

File: app/views/site/about.html.slim (hunk 1 above)
Comment (andylushman) [reply app/views/site/about.html.slim]: Added the classes to `about.sass`, which already existed"

File: app/views/site/about.html.slim (hunk 2 above)
Comment (andylushman) [reply app/views/site/about.html.slim]: Added to `about.sass`

File: db/structure.sql [hunk 4]
```diff
@@ -9,34 +9,6 @@ SET xmloption = content;
...
```
Comment (scpaye506) [inline db/structure.sql]: there should be no diff for this file

Comment (andylushman) [reply db/structure.sql]: Just fixed it 👍
